focus.el

```diff
@@ -220,7 +219,7 @@ if active."
     (focus-goto-thing (focus-bounds))))
 
 (defun focus-prev-thing (&optional n)
-  "Moves the point to the middle of the Nth previous thing."
```

Comment:
Flycheck-mode gave me a warning here. I think `Move` is the correct word to use since you have it inside other command. Like line [#L178](https://github.com/jcs-PR/Focus/blob/master/focus.el#L178).